tests/src/lib.rs

```diff
@@ -46,7 +64,7 @@ pub fn valid<T>(device: &wgpu::Device, callback: impl FnOnce() -> T) -> T {
 /// provided `should_fail` boolean.
 pub fn fail_if<T>(device: &wgpu::Device, should_fail: bool, callback: impl FnOnce() -> T) -> T {
     if should_fail {
-        fail(device, callback)
+        fail(device, callback, None)
```

Comment:
- [ ] todo: Let's expose this argument in `fail_if`. I don't see a reason not to migrate its callers now.